xc7/libraries/cells_xtra.xml

```diff
@@ -0,0 +1,1430 @@
+</module>
+
+<module name="IBUFDS_INTERMDISABLE">
+	<port name="I" type="input" width="1">Input p-side port connection. Connect directly to a top-level port in the design.</port>
+	<port name="IB" type="input" width="1">Input n-side port connection. Connect directly to a top-level port in the design.</port>
+	<port name="IBUFDISABLE" type="input" width="1">Disables input path through the buffer and forces to a logic High when USE_IBUFDISABLE is set to "TRUE" and this signal is asserted High. If USE_IBUFDISABLE is set to "FALSE" this input is ignored and should be tied to ground. This feature is generally used to reduce power at times when the I/O is idle.</port>
+	<port name="INTERMDISABLE" type="input" width="1">Disables input termination. This feature is generally used to reduce power at times when the I/O is idle.</port>
+	<port name="O" type="output" width="1">Buffer output representing the input path to the device.</port>
+	<attribute name="DIFF_TERM" type="STRING" default="FALSE" values="TRUE, FALSE">Enable the built-in differential termination.</attribute>
+	<attribute name="IBUF_LOW_PWR" type="STRING" default="TRUE" values="TRUE, FALSE">Allows a trade off of lower power consumption versus. highest performance when referenced I/O standards are used.</attribute>
+	<attribute name="IOSTANDARD" type="STRING" default="DEFAULT" values="See Data Sheet">Assigns an I/O standard to the element.</attribute>
+	<attribute name="USE_IBUFDISABLE" type="STRING" default="TRUE" values="TRUE, FALSE">Enables or disables the IBUFDISABLE feature. Generally used when it is not desirable to disable the input path in order to allow a read during write operation.</attribute>
+</module>
+
+<module name="ICAPE2">
+	<port name="CLK" type="input" width="1">Clock Input</port>
+	<port name="CSIB" type="input" width="1">Active Low ICAP Enable</port>
+	<port name="I" type="input" width="32">Configuration data input bus</port>
+	<port name="O" type="output" width="32">Configuration data output bus</port>
+	<port name="RDWRB" type="input" width="1">Read/Write Select input</port>
+	<attribute name="DEVICE_ID" type="HEX" default="0'h3651093" values="32'h03651093, 32'h036A2093, 32'h036A4093, 32'h036A6093, 32'h036BF093, 32'h036B1093, 32'h036B3093, 32'h036C2093, 32'h036C4093, 32'h036C6093, 32'h036DF093, 32'h036D1093, 32'h036D3093, 32'h036D5093, 32'h036D9093, 32'h0362C093, 32'h0362D093, 32'h0363B093, 32'h0364C093, 32'h0371F093, 32'h0372C093, 32'h0377F093, 32'h03627093, 32'h03628093, 32'h03631093, 32'h03636093, 32'h03642093, 32'h03647093, 32'h03656093, 32'h03667093, 32'h03671093, 32'h03676093, 32'h03680093, 32'h03681093, 32'h03682093, 32'h03687093, 32'h03691093, 32'h03692093, 32'h03696093, 32'h03702093, 32'h03704093, 32'h03711093, 32'h03722093, 32'h03727093, 32'h03731093, 32'h03747093, 32'h03751093, 32'h03752093, 32'h03762093, 32'h03771093, 32'h03782093">Specifies the pre-programmed Device ID value to be used for simulation purposes.</attribute>
```

Comment:
Hmm, this is strange, will have to check that a text object hasn't been lost by the parser

Reply:
Annoyingly, the parser was working correctly and the mistake is actually in the spec itself. I added a special case handler because no hex constant should ever start with 0'h.